docs/book/src/reference/metrics.md

```diff
@@ -3,6 +3,36 @@
 By default, controller-runtime builds a global prometheus registry and
 publishes a collection of performance metrics for each controller.
 
+## Exporting Metrics for Prometheus
+
+To export the metrics we will use the Prometheus Operator. Following the steps to do this setup.
+
+1. Install Prometheus Operator. You can follow up the steps in [Kube Prometheus](https://github.com/coreos/kube-prometheus) project.
```

Comment:
Probably better to use `https://github.com/coreos/kube-prometheus#installing`

Reply:
Closing as solved.